File: SEIST/polyglot/ocaml/seis_release_contract.ml
let package_path = "dist/seis-static.zip"
let supported_locales = ["tr"; "en"; "fr"; "it"; "de"; "es"; "ar"]
let reduced_motion_required = true
let live_upload_requires_target = true

let supports_locale locale =
  List.exists ((=) locale) supported_locales

File: SEIST/polyglot/julia/SeisReleaseContract.jl
module SeisReleaseContract

const PACKAGE_PATH = "dist/seis-static.zip"
const SUPPORTED_LOCALES = ["tr", "en", "fr", "it", "de", "es", "ar"]
const LIVE_UPLOAD_REQUIRES_TARGET = true

supports_locale(locale::AbstractString) = locale in SUPPORTED_LOCALES

end
